MyLibrary/Library/Controllers/BookController.cs

```diff
@@ -2,10 +2,12 @@
 // Copyright (c) Peretiatko Anastasiia. All rights reserved.
 // </copyright>
 
-using System;
 using BusinessLogic.LibraryModel;
 using BusinessLogic.LibraryService;
 using Microsoft.AspNetCore.Mvc;
+using System;
```

Comment:
commonly System namespaces are placed first.
https://dailydotnettips.com/automatically-place-the-system-directives-first-when-sorting-usings-option-in-visual-studio/

Reply:
Thank you for link, it helps me